README.rst

```diff
@@ -1,9 +1,27 @@
 |logo| Pootle
 =============
 
-|chat| |build| |health| |coverage| |requirements|
+.. image:: https://img.shields.io/gitter/room/translate/pootle.svg?style=flat-square
```

Comment:
Yes, it is sad. But we need this to work in PyPI too. I spent 2 hours trying to keep this and to PyPI to parse it without luck.